kong/plugins/basic-auth/api.lua

```diff
@@ -52,5 +52,29 @@ return {
     DELETE = function(self, dao_factory)
       crud.delete(self.basicauth_credential, dao_factory.basicauth_credentials)
     end
+  },
+  ["/basic-auth/"] = {
```

Comment:
Should we also use the plural form `/basic-auths` here as well to stay consistent with what we decided for `/key-auths`?